common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -769,58 +774,57 @@ class @Problem
     # params:
     #   'enable' is a boolean to determine enabling/disabling of buttons.
     #   'isFromCheckOperation' is a boolean to keep track if operation was initiated
-    #    from @check so that text of check button will also be changed while disabling/enabling
-    #    the check button.
+    #    from @submit so that text of submit button will also be changed while disabling/enabling
+    #    the submit button.
     if enable
       @resetButton
         .add(@saveButton)
         .add(@hintButton)
         .add(@showButton)
-        .removeClass('is-disabled')
```

Comment:
I thought the Lettuce test was still depending on this class present. Is that no longer true?